runtime/doc/diagnostic.txt

```diff
@@ -570,8 +570,8 @@ match({str}, {pat}, {groups}, {severity_map}, {defaults})
 
                  local s = "WARNING filename:27:3: Variable 'foo' does not exist"
                  local pattern = "^(%w+) %w+:(%d+):(%d+): (.+)$"
-                 local groups = {"severity", "lnum", "col", "message"}
-                 vim.diagnostic.match(s, pattern, groups, {WARNING = vim.diagnostic.WARN})
+                 local groups = { "severity", "lnum", "col", "message" }
+                 vim.diagnostic.match(s, pattern, groups, { WARNING = vim.diagnostic.WARN })
```

Comment:
This also needs to be changed in `runtime/lua/vim/diagnostic.lua` lines 1528-1529